specs/card/card.md

```diff
@@ -0,0 +1,71 @@
+# Card
+
+## Overview
+
+*Cards* are snapshots of content that are typically used in a group to present collections of related information. They can represent any quickly consumable information.
+
+### Features
+
+- **Sizing:** Users want to be able to set the width and height of a single card or a set of cards.
+
+- **Elevation:** Users may want to set different elevations of a single card or a set of cards based on context, state (such as hover), etc.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/card)
+
+**Cards are a building block used to compose examples like those below:
+
+| Examples | light | dark |
+| ----- | ----- | ----- |
+| favorite | ![](./images/favorites-card.png) | ![](./images/favorites-card-dark.png)
+| content | ![](./images/content-card.png) | ![](./images/content-card-dark.png) |
+| shopping | ![](./images/shopping-card.png) | ![](./images/shopping-card-dark.png) |
+
+---
+
+## Design
+
+### API
+
+*Component name:*
+- `fast-card`
+
+*Attributes:*
+- extends HTMLDivElement
```

Comment:
Just reminding you that we can't do this in a cross-browser way.

Reply:
Yes, though I believe that problem exists with CSS parts as well, correct? It seems like extending from the div element here makes the most sense long term. We can avoid doing that if it's too problematic.

Reply:
The problem is that the card wouldn't be usable at all in Safari when extending from div. However, adding parts only limits customization. The component will still function.

Reply:
Updated to remove this note.